### PR TITLE
Fixing column_format argument passing, #9402

### LINKED ISSUE #9402: DataFrame to_latex does not present or pass through column_format to formatter
## Expected Behaviour
`column_format` keyword arguments to `df.to_latex` should honour the relevant options available and implemented in `DataFrameFormatter` and it's `to_latex` method

The reason why this is annoying is that default `frmt` behaviour is to justify based on the column datatype. For cases where the index column/first column is numeric, these columns get right-aligned, which looks horrible.

## Versions
```
INSTALLED VERSIONS
------------------
commit: None
python: 2.7.6.final.0
python-bits: 64
OS: Linux
OS-release: 3.13.0-27-generic
machine: x86_64
processor: x86_64
byteorder: little
LC_ALL: None
LANG: en_GB.UTF-8

pandas: 0.15.1
nose: 1.3.4
Cython: 0.20.2
numpy: 1.9.1
scipy: 0.14.0
statsmodels: 0.5.0
IPython: 2.3.1
sphinx: 1.2.2
patsy: 0.3.0
dateutil: 2.3
pytz: 2014.10
bottleneck: None
tables: 3.1.1
numexpr: 2.4
matplotlib: 1.4.2
openpyxl: 1.8.6
xlrd: None
xlwt: 0.7.5
xlsxwriter: None
lxml: 3.3.5
bs4: 4.3.2
html5lib: None
httplib2: 0.9
apiclient: None
rpy2: None
sqlalchemy: None
pymysql: None
psycopg2: None
````

### PR DESCRIPTION
**Pull request by glyg**
closes #9402

### CODE REVIEW

File: pandas/tests/test_format.py [hunk 1]
```diff
@@ -2380,6 +2380,24 @@ def test_to_latex(self):
 """
         self.assertEqual(withoutindex_result, withoutindex_expected)
 
+    def test_to_latex_format(self):
+        # it works!
```
Comment (jreback) [inline pandas/tests/test_format.py]: add the issue number as the comment

File: pandas/core/frame.py [hunk 2]
```diff
@@ -1486,6 +1487,9 @@ def to_latex(self, buf=None, columns=None, col_space=None, colSpace=None,
 
         bold_rows : boolean, default True
             Make the row labels bold in the output
+        column_format : str, default None
+            The columns format as specified in LaTeX (e.g 'rcl'
+            for a 3 columns table)
```
Comment (jreback) [inline pandas/core/frame.py]: can you add a link the formats?

**Comment by jreback** [comment]
pls add a release note (bug fix section).

**Comment by jreback** [comment]
pls squash, then ping when green.

**Comment by glyg** [comment]
I'm in transit today, no internet. I'll do that back home tonight or tomorrow. Best

Le 31 août 2015 01:53:26 GMT+01:00, Jeff Reback <notifications@github.com> a écrit :
>pls squash, then ping when green.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/pydata/pandas/pull/10944#issuecomment-136224665

-- 
Envoyé de mon appareil Android avec K-9 Mail. Veuillez excuser ma brièveté.

**Comment by glyg** [comment]
Found a connection :) 

I messed up the rebase somewhat, so there's an extra useless commit, but for the rest I think I did what you asked.

I plan on fixing some more `to_latex` bugs this week, shall I keep on doing this on that branch and PR, or is it better to make a branch for each bug?

**Comment by jreback** [comment]
I'll squash, don't worry.

always make new features on separate branches. thxs

**Comment by jreback** [comment]
merged via https://github.com/pydata/pandas/commit/fb42766b3eb55155d67548d364672d254be7ffb6

thanks!

**Pull request closed without merging**